src/main/java/bio/terra/workspace/service/datareference/utils/DataReferenceValidationUtils.java

```diff
@@ -44,19 +45,17 @@ public String validateReference(
   }
 
   private DataRepoSnapshot validateDataRepoReference(
-      String reference, AuthenticatedUserRequest userReq) {
-    try {
-      DataRepoSnapshot ref = objectMapper.readValue(reference, DataRepoSnapshot.class);
-      if (!dataRepoService.snapshotExists(ref.getInstanceName(), ref.getSnapshot(), userReq)) {
-        throw new InvalidDataReferenceException(
-            "The given snapshot could not be found in the Data Repo instance provided."
-                + " Verify that your reference was correctly defined and the instance is correct");
-      }
-      return ref;
-    } catch (JsonProcessingException e) {
+      DataRepoSnapshot ref, AuthenticatedUserRequest userReq) {
+    if (ref.getInstanceName() == null || ref.getSnapshot() == null) {
       throw new InvalidDataReferenceException(
-          "Input could not be parsed as a Data Repo snapshot"
-              + " reference. The only valid fields are instanceName and snapshot.");
+          "Invalid Data Repo Snapshot identifier: "
+              + "instanceName and snapshot must both be provided.");
     }
```

Comment:
maybe use `StringUtils.isEmpty()` here to also look for the empty string

Reply:
I'm gonna go with `isBlank` because it will also catch whitespace-only